libraries/tools/kotlin-gradle-plugin-integration-tests/src/test/kotlin/org/jetbrains/kotlin/gradle/VariantAwareDependenciesIT.kt

```diff
@@ -287,13 +283,13 @@ class VariantAwareDependenciesIT : BaseGradleIT() {
         // variants chosen over the runtime ones when resolving a configuration with no required Usage:
         with(Project("simpleProject")) {
             setupWorkingDir()
-            gradleBuildScript().appendText("\ndependencies { compile 'org.jetbrains.kotlin:kotlin-compiler-embeddable' }")
+            gradleBuildScript().appendText("\ndependencies { implementation 'org.jetbrains.kotlin:kotlin-compiler-embeddable' }")
```

Comment:
To keep this test still checking [the requirement](https://github.com/JetBrains/kotlin/pull/3970/files#diff-e4cb0de25baada0e7d2426d8854b5d3addb949a3f3ed109836e198117f4ff79dR281-R283), I think we need to add a custom resolvable configuration that extends from `implementation` and has no explicit `Usage` attribute.